resiliency/service-to-component-resiliency.md

```diff
@@ -0,0 +1,120 @@
+In this QuickStart, you will run a microservice application that continuously persists and retrieves state via Dapr's state management API. When operations to the state store begin to fail, Dapr resiliency policies are applied.
+
+Visit [this](https://docs.dapr.io/operations/resiliency/resiliency-overview//) link for more information about Dapr resiliency.
+
+This quickstart includes one service:
+
+- Client service `order-processor` 
+- Redis component spec `statestore.yaml`
+- Resiliency spec `resiliency.yaml`
+
+### Run the client service with Dapr and resiliency enabled
+
+1. Navigate to the app directory, install dependencies, and run the service with resiliency enabled via the config.yaml: 
+
+### CSharp example:
+```bash
+cd ../state_management/csharp/sdk/order-processor
```

Comment:
nice idea adding a "drop in" resiliency policy to an existing quickstart.  This keeps the key concept at hand.